tests/cypress/support/commands.js

```diff
@@ -278,3 +280,43 @@ Cypress.Commands.add("getMany", (names) => {
     }
     return cy.wrap(values);
 });
+
+Cypress.Commands.add("createWithAPI", (url, values) => {
+    return cy.initApi().doApiRequest("POST", url, values).then(response => {
+        return response.body.id;
+    });
+});
+
+Cypress.Commands.add("updateWithAPI", (url, values) => {
+    cy.initApi().doApiRequest("PUT", url, values);
+});
+
+Cypress.Commands.add("initApi", () => {
+    if (api_token !== null) {
+        return api_token;
+    }
+
+    cy.request({
+        auth: {
+            'user': 'e2e_tests',
+            'pass': 'glpi',
+        },
+        method: 'POST',
+        url: '/apirest.php/initSession',
+    }).then((response) => {
+        return response.body.session_token;
+    });
+});
+
+Cypress.Commands.add("doApiRequest", {prevSubject: true}, (token, method, endpoint, values) => {
+    cy.request({
```

Comment:
Missing return?

Reply:
Indeed, not sure why it works without it...